SplitFiles/control.h

```diff
@@ -6,7 +6,7 @@
 // max stack depth
 static const unsigned int CONTROL_BUFFER_SIZE = 10;
 // length game runs for
-static const unsigned int MAX_TICKS = 8192;
+static const unsigned int MAX_TICKS = 4096;
```

Comment:
Gawd it is tedious at 8192 ticks

Reply:
LOL it's not a game yet, it's a series of POCs